README.md

```diff
@@ -8,8 +8,10 @@ How to
 
 ### Step 1: Install the plugin
 
+Clone the repo. In the root of the repo run:
+
 ```bash
-$ GOBIN=$PWD/build go install go.einride.tech/protoc-gen-go-aip-cli
+$ go install ./cmd/protoc-gen-go-aip-cli
```

Comment:
Good catch that the install path is wrong!

I agree that the docs are a bit weird, as I think you're right that the following steps do require you to clone the repo!

But can we perhaps keep the public install URL here (my suggestion should work I hope, just tried it) - and perhaps move the "Clone the repo" part later, just to make it easy for users to try it out on their own repo instead of trying out the complete example? If you get my point!

```suggestion
$ GOBIN=$PWD/build go install go.einride.tech/aip-cli/cmd/protoc-gen-go-aip-cli
```